### PR TITLE
use a better way to update react-native package

### PR DESCRIPTION
**Pull request by sunnylqm**

### CODE REVIEW

File: docs/Upgrading.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ version of React Native:
 
 ## 1. Upgrade the `react-native` dependency
 
-Note the latest version of the `react-native` npm package from here:
+Note the latest version of the `react-native` npm package from here(or use `npm info react-native` to check):
```
Comment (ide) [inline docs/Upgrading.md]: space between `here` and `(`

File: docs/Upgrading.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ latest version. Now, in a terminal run:
 $ npm install
 ```
 
+_NOTE:_ Some developers may use `react-native -v` to check current version. That's not correct.
+This shows the version of `react-native-cli`, which is a global command tool, but not `react-native`.
```
Comment (satya164) [inline docs/Upgrading.md]: nit: Have all sentences on the same line.

**Comment by satya164** [comment]
Since you're changing this file, can you also change the following,

    Open your `package.json` file and update the version of `react-native` under `dependencies` to the
    latest version. Now, in a terminal run:

    ```sh
    $ npm install
    ```

to following,

    Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.18`, in a terminal run:

    ```sh
    $ npm install --save react-native@0.18
    ```

File: docs/Upgrading.md [hunk 3]
```diff
@@ -15,17 +15,18 @@ version of React Native:
...
-Open your `package.json` file and update the version of `react-native` under `dependencies` to the
-latest version. Now, in a terminal run:
+Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.18`, in a terminal run:
 
 ```sh
-$ npm install
+$ npm install --save react-native@0.18
 ```
 
+_NOTE:_ Some developers may use `react-native -v` to check current version. That's not correct. This shows the version of `react-native-cli`, which is a global command tool, but not `react-native`.
```
Comment (satya164) [inline docs/Upgrading.md]: Might need a bit of rewording,

```
`react-native --version` doesn't give the version of the `react-native` package. The command `react-native` is provided by the `react-native-cli` package and `react-native --version` shows the version of the `react-native-cli` package.
```

Comment (brentvatne) [inline docs/Upgrading.md]: I wonder if a better approach here is to have the CLI `-v` flag output the following:

```
react-native-cli: 0.x.y
react-native: 0.a.b
```

And if we are not inside of a project directory:

```
react-native-cli: 0.x.y
react-native: n/a - not currently inside React Native project directory
```

Comment (sunnylqm) [inline docs/Upgrading.md]: That would be great!

File: docs/Upgrading.md [hunk 4]
```diff
@@ -15,17 +15,18 @@ version of React Native:
...
-Open your `package.json` file and update the version of `react-native` under `dependencies` to the
-latest version. Now, in a terminal run:
+Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.18`, in a terminal run:
 
 ```sh
-$ npm install
+$ npm install --save react-native@0.18
 ```
 
+`react-native --version` doesn't give the version of the `react-native` package. The command `react-native` is provided by the `react-native-cli` package and `react-native --version` shows the version of the `react-native-cli` package.
```
Comment (satya164) [inline docs/Upgrading.md]: Add __NOTE__ in the beginning :)

**Comment by satya164** [comment]
@sunnylqm Just squash your commits and then we can merge. Also if you could send a PR on what @brentvatne suggested, it will be great :)

**Comment by sunnylqm** [comment]
I'll try that.

File: docs/Upgrading.md [hunk 5]
```diff
@@ -15,17 +15,18 @@ version of React Native:
...
-Open your `package.json` file and update the version of `react-native` under `dependencies` to the
-latest version. Now, in a terminal run:
+Now install that version of `react-native` in your project with `npm install --save`. For example, to upgrade to the version `0.18`, in a terminal run:
 
 ```sh
-$ npm install
+$ npm install --save react-native@0.18
 ```
 
+ _NOTE:_ `react-native --version` doesn't give the version of the `react-native` package. The command `react-native` is provided by the `react-native-cli` package and `react-native --version` shows the version of the `react-native-cli` package.
```
Comment (satya164) [inline docs/Upgrading.md]: Remove the note

**Pull request merged**